### DISCUSSION TITLE
新手如何快速入门Obsidian+常见问题汇总？

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#17] [Q&A常见问题]
在这里写下你给新手的建议吧！有新手入门的文章/视频等，欢迎分享啊！！！

**Comment by makerjackie** [comment]
抛转引玉：
个人推荐把官方文档撸一遍，再配合论坛上搜索常见问题，就差不多了
* [**官方英文帮助文档**](https://publish.obsidian.md/help)
* [**中文帮助文档**-过时-待更新](https://jackiegeek.gitee.io/obsidian-docs/zh/Obsidian/)
* [**ob教程** @社区成员-落山鸡](https://publish.obsidian.md/chinesehelp)

推荐：
简明易懂，图文并茂的[少数派：玩转 Obsidian | 基础设置篇](https://sspai.com/post/63481)

**Reply by leozjr** [reply]
这个教程现在有密码了

**Reply by makerjackie** [reply]
密码是 obsidian （最近在展示 设置密码的功能，后面会取消掉这个密码） @rjzoel
